src/sagemaker/jumpstart/artifacts/payloads.py

```diff
@@ -0,0 +1,84 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module contains functions for obtaining example payloads for JumpStart models."""
+from __future__ import absolute_import
+from copy import deepcopy
+from typing import Dict, Optional
+from sagemaker.jumpstart.constants import (
+    DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
+    JUMPSTART_DEFAULT_REGION_NAME,
+)
+from sagemaker.jumpstart.enums import (
+    JumpStartScriptScope,
+)
+from sagemaker.jumpstart.types import JumpStartSerializablePayload
+from sagemaker.jumpstart.utils import (
+    verify_model_region_and_return_specs,
+)
+from sagemaker.session import Session
+
+
+def _retrieve_default_payloads(
+    model_id: str,
+    model_version: str,
+    region: Optional[str],
+    tolerate_vulnerable_model: bool = False,
+    tolerate_deprecated_model: bool = False,
+    sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
+) -> Optional[Dict[str, JumpStartSerializablePayload]]:
+    """Returns default payloads.
+
+    Args:
+        model_id (str): JumpStart model ID of the JumpStart model for which to
+            get default payloads.
+        model_version (str): Version of the JumpStart model for which to retrieve the
+            default resource name.
+        region (Optional[str]): Region for which to retrieve the
+            default resource name.
+        tolerate_vulnerable_model (bool): True if vulnerable versions of model
+            specifications should be tolerated (exception not raised). If False, raises an
+            exception if the script used by this version of the model has dependencies with known
+            security vulnerabilities. (Default: False).
+        tolerate_deprecated_model (bool): True if deprecated versions of model
```

Comment:
One note here: AWS style guide advises to avoid "deprecate" whenever possible. --> "For headings, tables, and similar applications where space is limited, use terms such as discontinued, replaced by, or no longer supported, or similar language as appropriate that clearly indicates what was changed." Might want to loop in legal for a quick approval here if that hasn't already been done.

Reply:
Resolving comment due to offline discussion:
"Since it's been around, it's probably fine! But probably also good for PMs to be aware" -- Judy Heflin